okrs/2019-q3.md

```diff
@@ -0,0 +1,27 @@
+## Objective: Projects that depend on IPLD are using new interfaces and specifications
+
+* Complete unixfsv2 implementation and relevant `iq` functionality (@mikeal)
+* Document `iq` and write a blog post (@mikeal)
+* Complete selector work necessary for GraphSync (@warpfork)
+* TODO: go-ipld-prime OKR's
+* TODO: schema OKR's
```

Comment:
JS schema parser is _nearly_ finalized, there's a bunch of TODOs scattered throughout to tighten it up plus there's some change proposals in limbo for schemas that remain unresolved ("implicit" vs "default", inline representation modifications for fields with parens). So "Command-line schema parser" would be an appropriate OKR I think. Something that just spits out JSON given an .ipldsch file.

Made good progress in JS on both validating data and some of the minor data transforms too. There's a lot more work there to fully cover everything and implement enough tests to verify it works on very complex data. But that could be picked up as an OKR for someone: "Working schema data validator in JS or Go" and "Working schema data transformer in JS or Go". Or combine the two (I found the two operations are so close they are most easily implemented in one primary path).